cluster/charts/crossplane/README.md

```diff
@@ -79,7 +79,9 @@ and their default values.
 | `nodeSelector` | Enable nodeSelector for Crossplane pod | `{}` |
 | `customLabels` | Custom labels to add into metadata | `{}` |
 | `customAnnotations` | Custom annotations to add to the Crossplane deployment and pod | `{}` |
+| `serviceAccount.create` | Create serviceaccount for crossplane. If false, a own serviceAccount must be exists. | `true` |
```

Comment:
There is a discussion here that you might want to take a look https://github.com/crossplane/crossplane/pull/2756#discussion_r766759008

Reply:
Thanks for reference the discussion. 

In my scenario, I also use a different existing service Account with a different name.

Please also note, having the option `create`, `labels`, `name`, `annotation` is quite common for helm charts and I would appreciate it to follow this.

* https://github.com/grafana/helm-charts/blob/76aabf5f44ec6912e67e41aa341dfbd74b861bfe/charts/grafana/values.yaml#L16-L23
* https://github.com/prometheus-community/helm-charts/blob/d5bb47544cc3c116fc4f173fce2121da5c270216/charts/kube-prometheus-stack/values.yaml#L153-L156
* https://github.com/bitnami/charts/blob/4b0ff6c80280821a1dd4ee1ca5a6b14bc9fc6c7c/bitnami/postgresql/values.yaml#L1153-L1167
* https://github.com/kubernetes/dashboard/blob/6e3ef1f98a6b8d41fd19b75bb476c6460d549a78/charts/helm-chart/kubernetes-dashboard/values.yaml#L315-L320

Reply:
How to control the labels / annotation for the ServiceAccount provided by the helm chart?

Reply:
You would create your own `ServiceAccount` with the annotations and labels and just give its name as a Helm parameter and there wouldn't be another `ServiceAccount` created by the chart.

Reply:
This is quite unhandly if ArgoCD is in use, since it would require 2 dedicated ArgoCD Application.

I can't get the blocker for adding the 2 additional helm parameters here to provide the functionally.